docs/source/modules/tutorial.rst

```diff
@@ -0,0 +1,653 @@
+   (which should be one level down from the original created folder/directory).
+   Once you are in this ``sample-model-submission`` repo,
+   run the command below  (note the period; this tells pip to install all the dependencies you will
+   need, a nice and handy way to do this). ::
+     pip install .
+   In Pycharm, you can check to make sure these dependencies were installed correctly
+   by going into ::
+     Pycharm -> settings (preferences on Mac) -> project: brainscore-brief -> project interpreter
+   where you will see a list of around 100 packages like ``toml``, ``xarray``, and
+   ``Keras-preprocessing``. *(Note: installing all the dependencies will take around 2-5 mins
+   on your machine, depending on the hardware/internet)*. A different IDE will most likely
+   have a similar feature, but this tutorial uses Pycharm.
+5. Congrats! You now have completed the hardest part of install.
+   Also remember before running the pip command, make sure to navigate
+   using terminal into the correct folder using the ::
+     cd sample-model-submission
+   command to ensure it is installed in the right place- otherwise you get error #1
+   in the ``Common Errors: Setup`` section. Feel free to explore the various
+   files and get a feel for them.
+
+That’s it! You have downloaded and retrieved all of the files you need to submit a model!
+Take a break and go get some lunch or some donuts. If you get an error that is not
+listed/resolved below, reach out to us at MIT and we can (most likely) help:
+
+- msch@mit.edu
+- mferg@mit.edu
+- cshay@mit.edu
+
+Submit a Model to Brain-Score.org Part 1: Preparing the Model
+=============================================================
+
+By now you should have the ``sample-model-submission`` repo cloned and
+the dependencies installed. It is now time to prepare your model to be
+submitted! In this part we will submit a standard, generic form of AlexNet
+(implemented in Pytorch) in order to get a feel for the submission process.
+In Part 3 I will show you how to submit a custom Pytorch model, which is
+most helpful for those that want to submit their own model.
+
+1. Navigate, using your IDE’s Project Window (usually the left side of the
+   screen that shows all the folders/files), into the
+   ``sample-model-submission/examples/pytorch.py`` Python file.
+   If you did the above steps correctly, you will be able to simply
+   hit run on this file and the "prepping" service will commence.
+   What does that mean? The code in this file downloads, prepares, and
+   "mock scores" your model on a benchmark of choice, in order to ensure
+   everything works correctly for the main Brain-Score site submission.
+   It is like a check: if all goes well running this code, then your model
+   is ready to submit to the site to be scored. (*Note: the first time running
+   this file will take a bit, because you have to download the model
+   (AlexNet in this case) weights/data from the site. This took about 15 mins
```

Comment:
"weights/data from the site." -> "weights as well as ImageNet validation images (for PCA initialization)."